src/components/top-bar/_top-bar.scss

```diff
@@ -1,18 +1,28 @@
+@import '~styles/variables';
+
 .player .top-bar {
   background: linear-gradient(0deg, rgba(0, 0, 0, 0) 0%, rgba(0, 0, 0, 0.6) 100%);
   color: #fff;
-  opacity: 0;
   visibility: hidden;
-  height: 0;
-  transition: 100ms opacity;
+  height: auto;
+  max-height: 0;
+  transform: translateY(#{-$top-bar-max-height}px);
```

Comment:
missing in bottom bar?

Reply:
No, Bottom is different in this case

Reply:
Actually, I removed the max-height in the top-bar. Topbar animates the translate and bottombar animates the max-height